demos/interactive_face_detection_demo/cpp/main.cpp

```diff
@@ -2,228 +2,311 @@
+DEFINE_string(mem, "", mem_msg);
+
+constexpr char mhp_msg[] = "path to an .xml file with a trained Head Pose Estimation model";
+DEFINE_string(mhp, "", mhp_msg);
+
+constexpr char mlm_msg[] = "path to an .xml file with a trained Facial Landmarks Estimation model";
+DEFINE_string(mlm, "", mlm_msg);
+
+constexpr char o_msg[] = "name of the output file(s) to save";
+DEFINE_string(o, "", o_msg);
+
+constexpr char r_msg[] = "output inference results as raw values";
+DEFINE_bool(r, false, r_msg);
+
+constexpr char show_msg[] = "(don't) show output";
+DEFINE_bool(show, false, show_msg);
+
+constexpr char show_emotion_bar_msg[] = "(don't) show emotion bar";
+DEFINE_bool(show_emotion_bar, false, show_emotion_bar_msg);
+
+constexpr char smooth_msg[] = "(don't) smooth person attributes";
+DEFINE_bool(smooth, false, smooth_msg);
+
+constexpr char t_msg[] = "probability threshold for detections. Default is 0.5";
+DEFINE_double(t, 0.5, t_msg);
+
+constexpr char u_msg[] = "resource [U]tilization graphs: -u cdm. "
+    "c - average [C]PU load, d - load [D]istrobution over cores, m - [M]emory usage";
+DEFINE_string(u, "", u_msg);
+
+void parse(int argc, char *argv[]) {
+    gflags::ParseCommandLineFlags(&argc, &argv, false);
+    if (FLAGS_h || 1 == argc) {
+        std::cout << "  \t [-h]                                        " << h_msg
+                  << "\n\t[--help]          print help on all arguments"
+                  << "\n\t  -i <INPUT>                                 " << i_msg
+                  << "\n\t  -m <MODEL FILE>                            " << m_msg
+                  << "\n\t[--bb_enlarge_coef <NUMBER>]                 " << bb_enlarge_coef_msg
+                  << "\n\t [-d <DEVICE>]                               " << d_msg
+                  << "\n\t[--dx_coef <NUMBER>]                         " << dx_coef_msg
+                  << "\n\t[--dy_coef <NUMBER>]                         " << dy_coef_msg
+                  << "\n\t[--fps <NUMBER>]                             " << fps_msg
+                  << "\n\t[--limit <NUMBER>]                           " << lim_msg
+                  << "\n\t[--loop]                                     " << loop_msg
+                  << "\n\t[--mag <MODEL FILE>]                         " << mag_msg
+                  << "\n\t[--mam <MODEL FILE>]                         " << mam_msg
+                  << "\n\t[--mem <MODEL FILE>]                         " << mem_msg
+                  << "\n\t[--mhp <MODEL FILE>]                         " << mhp_msg
+                  << "\n\t[--mlm <MODEL FILE>]                         " << mlm_msg
+                  << "\n\t[ -o <OUTPUT>]                               " << o_msg
+                  << "\n\t [-r]                                        " << r_msg
+                  << "\n\t[--show] ([--noshow])                        " << show_msg
+                  << "\n\t[--show_emotion_bar] ([--noshow_emotion_bar])" << show_emotion_bar_msg
```

Comment:
```suggestion
                  << "\n\t[--show_emotion_bar] ([--noshow_emotion_bar]) " << show_emotion_bar_msg
```
You should have space here so when it printed there was at least one space
That changes the alignment for all other messages